### ISSUE TITLE
Gradle 2.13 - ClassNotFoundException for org.gradle.logging

### ISSUE THREAD

**Issue by eoftedal** [nebula-plugins/gradle-lint-plugin#58]
```
...
Caused by: java.lang.ClassNotFoundException: org.gradle.logging
   at com.netflix.nebula.lint.StyledTextService.<init>(StyledTextService.groovy:17)
   at com.netflix.nebula.lint.plugin.LintGradleTask$1.lintFinished(LintGradleTask.groovy:48)
...
```
Looks like you are accessing some gradle internal features, and that those are not working properly

**Comment by eoftedal** [comment]
I would suggest that instead of failing if none can be found, you default to a dummy implementation  that just does non-colored output using standard output mechanisms.

**Comment by eoftedal** [comment]
I'm wondering if there's a bug here: https://github.com/nebula-plugins/gradle-lint-plugin/blob/master/src/main/groovy/com/netflix/nebula/lint/StyledTextService.groovy#L21
Shouldn't it say:
`factoryClass = Class.forName('org.gradle.logging.StyledTextOutputFactory')`
instead of 
`factoryClass = Class.forName('org.gradle.logging')`

**Comment by rspieldenner** [comment]
fixed in #61
